content/docs/user-guide/checkpoints.md

```diff
@@ -0,0 +1,425 @@
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace     │ -            │    9 │ 0.066135 │ 0.9792 │ 473987 │ 0.001 │ 0            │
+│ full_pipeline │ Apr 20, 2021 │    - │        - │      - │ 473987 │ 0.001 │ 0            │
+│ │ ╓ exp-e81d6 │ 01:30 PM     │    9 │ 0.066135 │ 0.9792 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 34b1d6f   │ 01:30 PM     │    8 │ 0.066629 │ 0.9783 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ c048966   │ 01:30 PM     │    7 │ 0.082436 │ 0.9747 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 2910f35   │ 01:30 PM     │    6 │  0.08161 │ 0.9749 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 255d3aa   │ 01:30 PM     │    5 │ 0.088654 │ 0.9716 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ c976c6f   │ 01:29 PM     │    4 │  0.13659 │ 0.9574 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 0bdcd86   │ 01:29 PM     │    3 │  0.16031 │ 0.9507 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 574bdc3   │ 01:29 PM     │    2 │  0.21992 │ 0.9345 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ d0a150d   │ 01:29 PM     │    1 │  0.33183 │ 0.9037 │ 473987 │ 0.001 │ 0            │
+│ ├─╨ 239affa   │ 01:29 PM     │    0 │  0.49894 │ 0.8481 │ 473987 │ 0.001 │ 0            │
+│ │ ╓ exp-263da │ 01:19 PM     │   19 │ 0.048277 │ 0.9854 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 0391730   │ 12:05 PM     │    8 │ 0.066629 │ 0.9783 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 92042bd   │ 12:05 PM     │    7 │ 0.082436 │ 0.9747 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ a557ace   │ 12:05 PM     │    6 │  0.08161 │ 0.9749 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 114d93f   │ 12:05 PM     │    5 │ 0.088654 │ 0.9716 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 3289b4b   │ 12:05 PM     │    4 │  0.13659 │ 0.9574 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ e32a023   │ 12:04 PM     │    3 │  0.16031 │ 0.9507 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ cc52edc   │ 12:04 PM     │    2 │  0.21992 │ 0.9345 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 6d2e1d9   │ 12:04 PM     │    1 │  0.33183 │ 0.9037 │ 473987 │ 0.001 │ 0            │
+│ ├─╨ d121d33   │ 12:04 PM     │    0 │  0.49894 │ 0.8481 │ 473987 │ 0.001 │ 0            │
+└───────────────┴──────────────┴──────┴──────────┴────────┴────────┴───────┴──────────────┘
+```
+
+We can also remove all of the experiments we don't promote to our Git workspace
+with the following command: `dvc exp gc --workspace --force`.
+
+## Adding checkpoints to Git
+
+When you terminate training, you'll see a few commands in the terminal that will
+allow you to add these changes to Git.
+
+```dvc
+To track the changes with git, run:
+
+        git add dvclive.json dvc.yaml .gitignore train.py dvc.lock
+
+Reproduced experiment(s): exp-263da
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+You can run the following command to promote your experiments to the Git
+workspace.
```

Comment:
"Git workspace" is not the best term. It should be like "to save your experiments to the Git
history."